xlogger/decorator.py

```diff
@@ -0,0 +1,43 @@
+import functools
+import time
+from xlogger.logger import logger
```

Comment:
How does this work with the cross-file logging ideology? Will this work with the same logger instance?